app/controllers/api/clients_controller.rb

```diff
@@ -0,0 +1,27 @@
+module API
```

Comment:
I'll modify this to make the namespacing only a single line (like you have it in the appointments controller).